src/main/java/org/skife/jdbi/v2/ClasspathStatementLocator.java

```diff
@@ -51,6 +51,7 @@ public static boolean looksLikeSql(String sql)
                || local.startsWith("create ")
                || local.startsWith("alter ")
                || local.startsWith("merge ")
+               || local.startsWith("replace ")
```

Comment:
what is that change? Is that part of the fix or some spurious thing that sneaked in? If the latter, I would prefer two separate commits (one to fix the bug, one for a feature change).